### ISSUE TITLE
Timezone switched when filling market

### ISSUE THREAD

**Issue by jfarid27** [AugurProject/augur-ui#2409]
I tried to crate a market and during creation, it was set with timezone UTC -4. After committing to MainNet it somehow switched to UTC -5. I'm not entirely sure why or how to recreate the issue but it seemed weird. Using Chrome 69.0.3497.100 with Metamask on a Linux machine.

**Comment by adrake33** [comment]
Internal tracking ticket: https://app.clubhouse.io/augur/story/17667/end-time-is-not-getting-set-properly-when-creating-a-market

**Comment by phoebemirman** [comment]
this is because of daylight savings time, it is utc offset differs by an hour depending on whether it is between march 11th and nov 4th.

**Comment by 0xpunk** [comment]
Why was this issue closed? Yes it is because of timezone but it is still a big problem that the UI shows a different timezone when entering the expiration, than the one that will actually be used!

At no point is this visible until after the market is created and it is too late to change it. I just got bit by this issue myself and now have no idea what to do, except kiss my staked REP and ETH goodbye...

When you go to create a market, the expiration field is labeled "EXPIRATION TIME (UTC -1)". Obviously this very strongly implies that you are entering a UTC -1 time. Only after you submit the tx to actually create the market can you see that it actually took your time and used UTC -2, so the market actually expires one hour earlier than intended!

If the app is going to adjust my time for DST why wouldn't that be done BEFORE submitting the market, when you still have a chance to add it correctly?? Just update the timezone in the label when user sets the date... or better yet just use UTC everywhere and avoid this nonsense altogether

Any advice on what to do with my broken market? It expires one hour too early now, can I assume that it should resolve Invalid and report it so? Now I am worried that someone with more REP can just say "nah, one hour is close enough" and dispute it...

Sorry for the long rant but needless to say this is a bit stressful! I feel like I just burned almost 9 REP and to add insult to injury, the validity bond too... FML LOL

**Comment by pgebheim** [comment]
We have some other tickets related to timezone handling re-design already scheduled, I'm gonna transfer this issue into the monorepo and add it to that epic.

Thanks for the report @0xpunk -- will add a link to the new issue...

**Comment by pgebheim** [comment]
Issue moved to [AugurProject/augur #1081](https://github.com/AugurProject/augur/issues/1081) via [**ZenHub**](https://www.zenhub.com/)

**Comment by pgebheim** [comment]
Didn't mean to close this one!
